src/list-items/list-item-label.html

```diff
@@ -60,10 +60,11 @@
         display: inline-block;
         padding: 3px 6px;
         border-radius: 6px;
-        color: var(--primary-background-color);
-        color: var(--tag-text-color);
-        background-color: var(--pr-status-color);
-        background-color: var(--tag-color);
+        color: var(--primary-background-color, --tag-text-color);
+        background-color: var(--pr-status-color, --tag-color);
+        overflow: hidden;
+        text-overflow: ellipsis;
+        max-width: 120%;
```

Comment:
Didn't got this fixed in the class it was used.

Is making a new type for this in list-item-label, also good for you?
I will then use max-width 100%, as this should always work. 

Maybe it was a bit shortsighted of me, as I saw no labels where broken by this change I thought it was ok.